holidayextras/brands
# Added body bg image to hx brand

#### What does this PR do? (please provide any background :trollface:) 

In the HX brand, I have taken the liberty of adding the branded background image to the body by default.
I have made the background-attachment fixed on `@screen-sm-min` and up, which I feel helps to create a subtle sense of 'depth' in pages.

#### What tests does this PR have?
n/a

#### How can this be tested?
Check out [this PR](https://github.com/holidayextras/pattern-library/pull/103) and view some stuff in Pattern Library – e.g. the page layout example on the _Blocks_ page.

#### Any tech debt?
Not that I am aware of.

#### Screenshots / Screencast
n/a

#### What gif best describes how you feel about this work?
![](http://i.giphy.com/AC6kJUjgqERxe.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
